src/Pages/Login/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import LoginArticle from "./Components/LoginArticle";
+import RegisterArticle from "./Components/RegisterArticle";
+import ErrorTextBox from "./Components/ErrorTextBox";
+import { Config } from "./Components/Config";
+
+class LoginInput extends Component {
+  constructor() {
+    super();
+    this.state = {
+      errorMessage: "",
+      userID: "",
+      loginPassword: "",
+      email: "",
+      regPassword: "",
+      ToggleAutoLogin: false,
+      ToggleSubcriptEmail: false
```

Comment:
- 변수명은 소문자로 시작할 수 있도록 해주세요
- 오타 수정해주세요
```suggestion
      toggleAutoLogin: false,
      toggleSubscriptEmail: false
```

Reply:
넵 다음부터는 주의하겠습니다